webapp/src/Controller/API/TeamController.php

```diff
@@ -49,6 +49,14 @@ public function __construct(
         parent::__construct($entityManager, $dj, $config, $eventLogService);
     }
 
+    private function mask_ranking(?Response $normalResponse, string $entity="team"): ?Response
```

Comment:
Formatting as reported by phpcs, but I would also call the method `maskRanking`

Reply:
I've removed the function now.